index.html

```diff
@@ -82,20 +92,56 @@
         {{partial 'pass'}}
       {{else}}
         <div id="moveInput">
-        <p>        
-          <button {{action "submitMove" "nw" }}>nw</button>
-          <button {{action "submitMove" "n" }}>n</button>
-          <button {{action "submitMove" "ne" }}>ne</button>
+        <p>      
+          {{#if nw}}  
```

Comment:
Just an optimization, but you could DRY all this up using a helper such that each of these if/else blocks could be {{ moveButton "nw" }} and the like.